### DISCUSSION TITLE
How to draw a straight line

### DISCUSSION THREAD

**Discussion by AstroFuyuko** [Stellarium/stellarium#2520] [Q&A] [answered]
Is there a way that I can draw a straight line?
I tried StelPainter Class like this,

`StelPainter.drawLine2d(0,0,100,100);`

 but it was below error. 
`"script error: \"ReferenceError: Can't find variable: StelPainter\"`

I know a way that I can draw a line using "constellationsart.fab" and "constellationship.fab"
I want to know how to use scripting (StelPainter class? or other  class? )

If there is a way that I can draw a straight line using HIP number, 
I want to know it.

![straightline](https://user-images.githubusercontent.com/109458605/179451579-7cd9f9cc-e2ed-4cfd-bc7d-e90efc767d7b.png)

Does anyone know it?

 
Thanks very much
Fuyuko

**Comment by alex-w** [comment, marked as answer]
This is not possible.

See [scripting ](https://stellarium.org/doc/0.22/scripting.html) docs for details

**Reply by AstroFuyuko** [reply]
Oh, This is not possible.
I got it.
Thank you for your replying.
I hope this function will be developed someday.
